<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="InputTab">
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeWidget" name="treeData">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="tabKeyNavigation">
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>1</number>
          </property>
          <column>
           <property name="text">
            <string>Data</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableCommonAttrs">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Common Variable</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableVariableAttrs">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Variable</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="DataPlot">
       <attribute name="title">
        <string>Data</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="FitTab">
       <attribute name="title">
        <string>Fit</string>
       </attribute>
       <widget class="QTextEdit" name="fitFunc">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>451</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="startParams">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>261</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>114</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fit function f(p,x):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>98</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Start parameter</string>
        </property>
       </widget>
       <widget class="QComboBox" name="xSelector">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>250</y>
          <width>111</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="loadFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="loadFile">
   <property name="text">
    <string>Load file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
